mix.exs

```diff
@@ -34,6 +34,8 @@ defmodule Ecto.Mixfile do
 
   defp deps do
     [{:poolboy, "~> 1.4"},
+     {:sbroker, "~> 0.6", github: "fishcakez/sbroker", branch: "gen"},
```

Comment:
> OR even push postgrex and mariaex and friends to use it too.

Before we do that I think we should see if the "smart" NoSQL connections perform better than the "dumb" SQL connections under network partition. What's good for `poolboy` might be bad for `sbroker` and vice versa.